src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.Setup.cs

```diff
@@ -81,9 +82,9 @@ public MergeTestBlockchain(IMergeConfig? mergeConfig = null)
             {
                 GenesisBlockBuilder = Core.Test.Builders.Build.A.Block.Genesis.Genesis
                     .WithTimestamp(UInt256.One);
-                Signer = new Eth2Signer(MinerAddress);
                 BlockConfirmationManager = new BlockConfirmationManager();
                 MergeConfig = mergeConfig ?? new MergeConfig() { Enabled = true, TerminalTotalDifficulty = "0" };
+                Signer = new Eth2Signer(MergeConfig.BlockAuthorAccount != "" ? new Address(MergeConfig.BlockAuthorAccount) : Address.Zero);
```

Comment:
We could remove Eth2Signer class and pass it to MergeSealEngine. Let's rename BlockAuthorAccount to FeeReciepient. 
Change this check MergeConfig.BlockAuthorAccount != "" to string.IsNullOrWhitespace